tests/test-qt.cpp

```diff
@@ -1068,7 +1068,6 @@ test_logout_sync ()
     QStringList discharges;
     for (int i = 0; d[i] != NULL; i++)
         discharges << d[i];
-    qDebug() << discharges;
```

Comment:
Why did you remove this?

Reply:
This an unrelated fix, I really should have put in a separate PR - https://github.com/canonical/snapd-glib/pull/183